src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+		submitNoSamplesPane.add(checkBoxSubmitFailingSamples);
+		checkBoxSubmitFailingSamples.setSelected(true);
+		checkBoxSubmitFailingSamples.setToolTipText("Submt all non-AC (Yes) submissions");
+
+		judgingTypesButtonGroup.add(submitAllJudgingTypesRadioButton);
+		submitAllJudgingTypesRadioButton.setSelected(true);
+		judgingTypesButtonGroup.add(submitSelectedJudgingTypeRadioButton);
+		
+		submitNoSamplesPane.add(submitAllJudgingTypesRadioButton);
+		submitNoSamplesPane.add(submitSelectedJudgingTypeRadioButton);
+		
+		leftSide.setPreferredSize(new Dimension(230, 230));
+		leftSide.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		leftSide.setWheelScrollingEnabled(true);
+		splitPane.setLeftComponent(leftSide);
+
+		// centerPane2.add(LogPanel, BorderLayout.SOUTH);
+		JScrollPane logScrollPanel = new JScrollPane(LogPanel);
+		logScrollPanel.setPreferredSize(new Dimension(60, 60));
+		logScrollPanel.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		logScrollPanel.setWheelScrollingEnabled(true);
+		splitPane.setRightComponent(logScrollPanel);
+		
+		
+		selectedNosLabel.setToolTipText("None selected");
+		selectedNosLabel.addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseClicked(MouseEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		submitNoSamplesPane.add(selectedNosLabel);
+		JButton selectJudementTypesButton = new JButton("Select");
+		selectJudementTypesButton.setToolTipText("Select Judgement Types");
+		selectJudementTypesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		
+		submitNoSamplesPane.add(selectJudementTypesButton);
+		
+		JLabel lblNewLabel_2 = getWhatsThisOne();
+		submitNoSamplesPane.add(lblNewLabel_2);
+		
+		JPanel submitNoSamplesPane_1 = new JPanel();
+		FlowLayout flowLayout_2 = (FlowLayout) submitNoSamplesPane_1.getLayout();
+		flowLayout_2.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane_1.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
+        checkBoxProblems.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxProblems.setToolTipText("Select Problems");
+        checkBoxProblems.setSelected(true);
+        submitNoSamplesPane_1.add(checkBoxProblems);
+        
+        submitAllProblemsRadioButton = new JRadioButton("Submit All");
+        submitAllProblemsRadioButton.setSelected(true);
+        submitNoSamplesPane_1.add(submitAllProblemsRadioButton);
+        
+        submitSelectedProblemsRadioButton = new JRadioButton("Submit Selected");
+        submitNoSamplesPane_1.add(submitSelectedProblemsRadioButton);
+        
+        selectedProblemsLabel = new JLabel("None selected");
```

Comment:
I think it would be much clearer if the text of the above label was contained within parentheses.  As it is, what I see is:

![image](https://github.com/pc2ccs/pc2v9/assets/9256691/6726b509-6fa5-4d59-8148-560317735427)


It's not at all clear that the "None Selected" text is (somehow) related to the `Submit Selected` radio button (as opposed, for example, to the `Select` button on the other side, or to some other "stand-alone" information).  If the GUI was (something like):

```
<radiobutton> Submit Selected (None selected)  <Button>
```
I think it would be easier to understand.

This would of course require changing more than just the above JLabel constructor; the parentheses would need to be added whenever the contents of the label changed.    For that reason it might be easier to consider a slightly different design -- for example, a "Show Selected" button.